libraries/SITL/SIM_Vicon.cpp

```diff
@@ -204,6 +204,53 @@ void Vicon::update_vicon_position_estimate(const Location &loc,
             NULL, 0);
         msg_buf[msg_buf_index].time_send_us = time_send_us;
     }
+
+    // TO DO add Confidence Level
```

Comment:
I guess these three lines of code are from Thien's script?  Let's remove them 'cuz I don't think they are relevant here.

Reply:
Fixed. I had left them in as a stretch goal for figuring out how to simulate the correlation coefficients.